.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -0,0 +1,46 @@
+{
+  // ...
+}
+```
+
+## Expected Behavior
+What did you expect to happen?
+
+## Actual Behavior
+What actually happened? Include any error messages or output.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)